CONTRIBUTING.md

```diff
@@ -376,7 +376,7 @@ will cause a broken ToC, and should be corrected to:
 
 ### Contributor T-shirt
 
-If your Pull Request to Kong/docs.konghq.com was accepted and fixes a bug, adds
+If your contribution to this repo was accepted and fixes a bug, adds
```

Comment:
I don't think saying "this repo" is right, we should enumerate or list criteria which repositories are considered acceptable.

Reply:
I got confused and thought this was in kong/kong instead of kong/docs.konghq.com. I think you can ignore this.